klippa/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "klippa"
+version = "0.1.0"
+edition = "2021"
```

Comment:
We should include the same fields we include in the other crates in this repo, take a look at those `Cargo.toml` as an example.